6-task-queueing/Gemfile

```diff
@@ -22,7 +22,7 @@ gem "omniauth-google-oauth2"
 gem "gcloud"
 # [END gcloud]
 # [START google_api_client]
-gem "google-api-client"
+gem "google-api-client", "~>0.8.0"
```

Comment:
@remi I had to add this due to gcloud <-> google-api-client incompatibilities that was forcing the ancient gcloud version